languoids/tree/grea1241/md.ini

```diff
@@ -6,9 +6,10 @@ level = family
 [classification]
 sub = **hh:hv:Manoharan:Andamanese**
 familyrefs = 
-	**hh:hv:Abbi:Andamanese**
+	**hh:hv:Abbi:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**
```

Comment:
Note: The convention for lists is to put each item in a separate line. So this should be
```
        **hh:hv:Abbi:Andamanese**
        **hh:typ:ComrieZamponi:Great-Andamanese**
```